package.json

```diff
@@ -8,9 +8,12 @@
   },
   "scripts": {
     "start": "react-app-rewired start",
+    "start:debugging": "node --inspect-brk node_modules/react-app-rewired/scripts/start.js",
```

Comment:
It seems that @Artoria2e5 did more. Is it okay to cherry-pick this line to master?